README.md

```diff
@@ -18,6 +18,25 @@ then create a new document
     doc = Govspeak::Document.new "^Test^"
     puts doc.to_html
 
+## Changes appearing across GOV.UK
+
+Some additional steps or considerations are needed to ensure changes to govspeak cascade across GOV.UK a holistic way.
+
+Once govspeak has been updated and version incremented then: 
+- [`govuk_publishing_components` govspeak](https://components.publishing.service.gov.uk/component-guide/govspeak) will also need updating
+- Publishing apps, including [content-publisher](https://github.com/alphagov/content-publisher) & [whitehall]( 
+https://github.com/alphagov/whitehall) also use govspeak, these apps will need to be released with the new govspeak version present.
+
+Also, consider if:
+- [whitehall]( 
+https://github.com/alphagov/whitehall) needs updating (as custom govspeak changes are present)
+- [govuk-content-schema](https://github.com/alphagov/govuk-content-schemas) needs updating
+- [govpspeak-preview](https://github.com/alphagov/govspeak-preview) is worth updating
+
+Any pages that use govspeak to generate Content will need to *republished* in order for the new changes to be reflected.  
+
+- Data Labs can help idenify which pages need updating by completing [this form](https://docs.google.com/forms/d/e/1FAIpQLSeHHWiOcRfj10-tn42uqbv7D8tYiyPUKLA4lPlkxTofr5T6zg/viewform?gxids=7628) and [#govuk-2ndline](https://docs.publishing.service.gov.uk/manual/2nd-line.html) can help with republishing.
+
```

Comment:
Not sure about this section (or if to remove entirely)

This form link for Data-labs is likely to rot, email is not desirable, wondering if there is a data-labs homepage or link within docs?

Reply:
I'm pretty sure this is standard data labs operations so that form is probably the most appropriate thing to put here. It might be worth asking DL about this to clarify?

Reply:
Np, thanks for this - I prefer this wiki link, I think it makes more sense - I've updated 🙌🏼